dome.py

```diff
@@ -146,46 +146,40 @@
 #Not Tested 
 # option 1
 # def my_callback(channel):
```

Comment:
I think the channel in def my_callback(channel) needs to be the GPIO pin. In this case, channel should be GPIO.input(37). This could be made into a variable to make things clean; e.g.,
IR_home = GPIO.input(37)
def my_callback(IR_home):

function could also be called something else, but that's not important right now.